docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -767,17 +767,47 @@ The procedure to investigate it is the same as the one for [`MimirSchedulerQueri
 
 This alert fires if queries are piling up in the query-scheduler.
 
-The size of the queue is shown on the `Queue length` dashboard panel on the `Mimir / Reads` (for the standard query path) or `Mimir / Remote Ruler Reads`
+#### Dashboard Panels
+
+The size of the queue is shown on the `Queue Length` dashboard panel on the [`Mimir / Reads`](https://admin-ops-eu-south-0.grafana-ops.net/grafana/d/e327503188913dc38ad571c647eef643) (for the standard query path) or `Mimir / Remote Ruler Reads`
 (for the dedicated rule evaluation query path) dashboards.
 
-How it **works**:
+The `Latency (Time in Queue)` is broken out in the dashboard row below by the "Expected Query Component" -
+the scheduler queue itself is partitioned by the Expected Query Component for each query,
+which is an estimate from the query time range of which component the querier will utilize to fetch data
+(ingester, store-gateway, both, or unknown).
+
+The row below shows peak values for `Query-scheduler <-> Querier Inflight Requests`, also broken out by query component.
+This shows when the queriers are saturated with inflight query requests
+as well as which query components are being utilized to service the queries.
+
+#### How it Works
 
 - A query-frontend API endpoint is called to execute a query
 - The query-frontend enqueues the request to the query-scheduler
 - The query-scheduler is responsible for dispatching enqueued queries to idle querier workers
-- The querier runs the query, sends the response back directly to the query-frontend and notifies the query-scheduler that it can process another query
+- The querier fetches data from ingesters, store-gateways, or both, runs the query against the data,
+  sends the response back directly to the query-frontend and notifies the query-scheduler that it can process another query.
```

Comment:
```suggestion
  Then, it sends the response back directly to the query-frontend and notifies the query-scheduler that it can process another query.
```

Reply:
We can break this into two sentences to avoid a run-on.

Also, I know the other bullets aren't a part of this PR, but we might consider adding punctuation to remain consistent.